src/index.css

```
@@ -288,171 +345,221 @@
-     Three tiers of visual weight: card > panel > input/subtle
-
-     CRITICAL RULE: Never apply `overflow: hidden` to a parent 
-     of a glass surface. The `::before` dome highlight and rim 
-     lighting require `overflow: visible` on all ancestors. If 
-     clipping is needed for layout reasons, apply it to an 
-     inner wrapper that does not contain the glass surface.
-     ============================================================ */
+  /* Canonical layout primitives. Feature components choose these roles; tokens decide color. */
+  .page-detail {
+    @apply themed-scrollbar flex h-full flex-col overflow-y-auto;
+    background-color: rgb(var(--surface-page));
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)